### ISSUE TITLE
ESLint 9 invalid options

### ISSUE THREAD

**Issue by RudyTheDev** [openstreetmap/iD#10225]
It seems that ESLint 9 has deprecated/changed some configuration stuff. My IDE (JetBrains Rider) reports:

````
Error: Invalid Options:
- Unknown options: reportUnusedDisableDirectives, resolvePluginsRelativeTo, rulePaths, useEslintrc, extensions, ignorePath
- 'extensions' has been removed.
- 'resolvePluginsRelativeTo' has been removed.
- 'ignorePath' has been removed.
- 'rulePaths' has been removed. Please define your rules using plugins.
- 'reportUnusedDisableDirectives' has been removed. Please use the 'overrideConfig.linterOptions.reportUnusedDisableDirectives' option instead.
````

These come from `processOptions (node_modules\eslint\lib\eslint\eslint-helpers.js:839:15)`

I am not familiar enough with any of this lint stuff to say more. These errors are not present on an earlier v8.57.0. So I am assuming some config files need to be updated? It's not saying where it's reading stuff from, but the IDE says it looks for `.eslintrc.*` files. These also seem to be more like warnings than errors because they don't stop the project being built/tested. I am not 100% sure this isn't just my IDE doing something weird.

**Comment by seishudev** [comment]
same issue

**Comment by k-yle** [comment]
This was fixed in #10406
